app/views/marketing/index.html.erb

```diff
@@ -4,10 +4,12 @@
   <div class="content-wrapper-narrow">
     <h2><%= t(".heading") %></h2>
     <p class="hero__subheading"><%= t(".subheading") %></p>
-    <form action="">
-      <input type="text">
-      <input type="submit">
-    </form>
+
+    <%= simple_form_for @registration, url: new_registration_path, method: :get do |f| %>
```

Comment:
In this case, yes.

I've extracted a partial for the sake of clarifying the form's purpose in https://github.com/thoughtbot/freshfoodconnect/pull/8/commits/80d8c2b3f66eb690ec897f8b2f6d2ff20218bb8d